src/SpinModelRegion.cpp

```diff
@@ -71,6 +71,11 @@ namespace geopm
         m_big_o = big_o_in;
     }
 
+    void SpinModelRegion::set_atom(std::function<void(void)>run_atom)
```

Comment:
std::function<void(void)> run_atom

There's a space missing between the std:function declaration and the name.

Reply:
kk